README.md

```diff
@@ -77,6 +77,21 @@ yourlogger.addHandler(airbrake.AirbrakeHandler())
 ```
 _by default, the `AirbrakeHandler` only handles logs level ERROR (40) and above_
 
+#### Additional Options
+More options are available to configure this library. Similar to setup, you can do this with environment variables
+```
+export AIRBRAKE_HOSTNAME=sassbox-101.prod.api
```

Comment:
Why create a potentially different env var when on some systems `$HOSTNAME` is already set?